mock/mock.spec

```diff
@@ -67,6 +66,7 @@ BuildRequires: python%{python3_pkgversion}-pylint
 %endif
 BuildRequires: python%{python3_pkgversion}-rpm
 BuildRequires: python%{python3_pkgversion}-rpmautospec-core
+BuildRequires:  systemd-rpm-macros
```

Comment:
I still don't understand why we are moving this logic from `mock-filesystem` to `mock-core-configs`, at least documentation is needed - but it seems wrong :shrug:

Reply:
+1

Reply:
thanks for noticing, I did this firstly in mock-core-config but then moved it to mock-filesystem. This is just a remnant I forgot to remove